python/ray/tune/tests/test_experiment_analysis.py

```diff
@@ -10,10 +10,60 @@
 import pandas as pd
 
 import ray
-from ray.tune import run, sample_from, Analysis
+from ray.tune import run, Trainable, sample_from, Analysis
 from ray.tune.examples.async_hyperband_example import MyTrainableClass
 
 
+class ExperimentAnalysisInMemSuite(unittest.TestCase):
+    scores_list = [5, 4, 2, 9, 7, 0, 1, 8, 6, 3]
```

Comment:
move this into `testCompareTrials` and pass into `tune.run(config={"scores": scores}`, then set `self.scores` in `_setup`, think that should work